libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.spec.ts

```diff
@@ -306,16 +306,16 @@ describe('ModalWrapperComponent', () => {
         await TestHelper.waitForResizeObserver();
 
         const ionToolbarElement = spectator.query('ion-toolbar');
-        const pageProgressAsIonToolbarChild = ionToolbarElement.querySelector(
-          'kirby-page-progress'
-        );
+        let pageProgressAsIonToolbarChild = ionToolbarElement.querySelector('kirby-page-progress');
         expect(pageProgressAsIonToolbarChild).not.toBeNull();
 
         embeddedComponent.showPageProgress = false;
         spectator.detectChanges();
 
-        pageProgress = pageProgressAsIonToolbarChild.querySelector(':scope > kirby-page-progress');
-        expect(pageProgress).toBeNull();
+        pageProgressAsIonToolbarChild = ionToolbarElement.querySelector(
+          ':scope > kirby-page-progress'
+        );
```

Comment:
Not sure I understand why this query is different from the first?

Reply:
There is no difference. The build server complained about linting - and `ionToolbarElement` was a const before and so it also complained about reassigning the variable